From fb6464015fa31c5e4a402f772cfdc766f39983cb Mon Sep 17 00:00:00 2001
From: Lin Zhang <zhanglin0425@thundersoft.com>
Date: Fri, 14 Feb 2014 16:14:30 +0800
Subject: [PATCH 3/3] defconfig: msm: Add CONFIG_IMX134_13P1BA833D

Add CONFIG_IMX134_13P1BA833D=y to the defconfig file of msm8610

Change-Id: Ibe26ad5d4657cdb505325360e00b62a710f208fc
Signed-off-by: Lin Zhang <zhanglin0425@thundersoft.com>
---
 arch/arm/configs/msm8610_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/msm8610_defconfig b/arch/arm/configs/msm8610_defconfig
index 1ca7ed6..91ad23d 100644
--- a/arch/arm/configs/msm8610_defconfig
+++ b/arch/arm/configs/msm8610_defconfig
@@ -318,6 +318,7 @@ CONFIG_OV8825=y
 CONFIG_s5k4e1=y
 CONFIG_HI256=y
 CONFIG_OV12830=y
+CONFIG_IMX134_13P1BA833D=y
 CONFIG_IMX134=y
 CONFIG_MSM_CAMERA_SENSOR=y
 CONFIG_MSM_EEPROM=y
-- 
1.7.4.1

